Scripts/datahandling/zonedata.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, data_dir: str, zone_numbers: numpy.array):
         parkdata = read_csv_file(data_dir, ".prk", self.zone_numbers, dtype)
         self.externalgrowth = read_csv_file(
             data_dir, ".ext",
-            all_zone_numbers[all_zone_numbers.searchsorted(external[0]):],
+            all_zone_numbers[all_zone_numbers.searchsorted(external[0]):all_zone_numbers.searchsorted(external[1],side='right')],
```

Comment:
This line is a bit long, but I guess this is otherwise the most concise way of doing this.